ansible/webscale.yml

```diff
@@ -9,8 +9,15 @@
         - always
   roles:
     - role: operators
+      when: operators is defined and operators | length > 0
       tags: always
 
+    - role: label_node
```

Comment:
Should the labeling of nodes occur before installing any operators?

Reply:
I don't thin it matters, since the operator install is just a subscription. We actually configure local-storage/prom separately.